lerna-util-sequence/src/main/scala/lerna/util/sequence/SequenceFactoryWorker.scala

```diff
@@ -185,7 +194,9 @@ private[sequence] final class SequenceFactoryWorker(
     case msg: GenerateSequence =>
       // no reply
       logger.warn(
-        "Pending generate sequence because reserving sequence: current max reserved: {}, next sequence value: {}",
+        "sequenceId=[{}], sequenceSubId=[{}]: Pending generate sequence because reserving sequence: current max reserved: {}, next sequence value: {}",
```

Comment:
SequenceFactoryWorker は次のようなメッセージを出力します:
```
15:56:25.950    WARN    lerna.util.sequence.SequenceFactoryWorker$      -       -       dummy   sequenceId=[test-sequence-id], sequenceSubId=[test]: Pending generate sequence because reserving sequence: current max reserved: 13, next sequence value: 23
15:56:36.267    WARN    lerna.util.sequence.SequenceFactoryWorker$      -       -       dummy   sequenceId=[737f4377-27f1-4341-9521-d3f41ef7d483], sequenceSubId=[-]: Pending generate sequence because reserving sequence: current max reserved: 1000, next sequence value: 1009
```